### ISSUE TITLE
Further Investigation needed for acroipm2.adobe.com

### ISSUE THREAD

**Issue by xRuffKez** [hagezi/dns-blocklists#963]
I need more Information for acroipm2.adobe.com

Must origin from Adobe Reader.

It loads zip files but can not be Updates for the Software itself.

Things i found so far:
https://community.adobe.com/t5/acrobat-reader-discussions/adobe-reader-keeps-downloading-http-acroipm-adobe-com-10-rdr-enu-win-nooem-none-message-zip/m-p/3543988

**Comment by hagezi** [comment]
No reader updates, the ZIP contains certificates that are updated every time the reader is started.

"_You can control this behaviour by selecting "Preferences" from the "Edit" menu and then going to the "Trust Manager" section. Here you can enable/disable loading these certificates, control whether Reader will ask you before updating and manually update now._"

**Comment by pZetan** [comment]
@hagezi I suspect this allow rule might need to be reviewed again. Should i create a new issue?

My review was using (AcroRdrDCx642600121662_MUI.exe) - https://get.adobe.com/reader/enterprise/

The embedded certificates are part of the package signature, not AATL/trust-list certificate updates.

Adobe Acrobat/Reader IPM and auxiliary app-control asset host (acroipm2.adobe.com). Serves signed message manifests, launch assets, launch/ping image bundles, experiments, upsell/marketing assets, and ARM-upsell-related packages. Captured payloads do not show any ProductVersions.msi, AATL/trust-list updates, core update installers, activation, login, or licensing.

**Comment by hagezi** [comment]
The domain is neither blocked nor unblocked. It does not appear in any source, but it does appear on a few allowlists.
